Changelog.md

```diff
@@ -5,6 +5,7 @@ Compiler Features:
  * SMTChecker: Support ABI functions as uninterpreted functions.
 
 Bugfixes:
+ * Parser: Mark the empty version pragma as invalid (``pragma solidity;``).
```

Comment:
Could remove this given it was an error previously too, just a different one.

Reply:
I'd vote for removing